### PR TITLE
Add 'Discover Cluster Providers' command

### PR DESCRIPTION
**Pull request by fbricon**
You can search for extensions contributing cluster providers in the extension gallery, by clicking on the `Discover Cluster Providers` button (also available via the command palette):

Those extensions must be labeled `kafka-provider`.

### CODE REVIEW

**Comment by fbricon** [comment]
We should see  at least one 3rd party provider next week ;-)

File: docs/Explorer.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ Multiple delete is not supported for the moment. See [issue 107](https://github.
...
 To delete a consumer group, it first must be stopped, else the `Delete` action will report an error.
 
+## Discover new cluster providers
+
+You can search for extensions contributing cluster providers in the extension gallery, by clicking on the `Discover Cluster Providers` button (also available via the command palette):
+
+![Discover Cluster Providers](assets/kafka-explorer-discover-providers.png)
+
+Those extensions must be labeled `kafka-provider`.
+
```
Comment (angelozerr) [inline docs/Explorer.md:42]: Perhaps it should use tagged instead of labeled and add a package.json sample with this tag.

Comment (angelozerr) [reply docs/Explorer.md:42]: Do you think you could add a screenshot here with the result of click on `Discover Cluster Providers` icon.

Comment (fbricon) [reply docs/Explorer.md:47]: not until there are providers available

Comment (fbricon) [reply docs/Explorer.md:47]: > Perhaps it should use `tagged `instead of `labeled `and add a package.json sample with this tag.

fixed

**Comment by angelozerr** [comment]
Thanks @fbricon !

**Pull request merged**